In [18]:
import pandas as pd
import numpy as np
from scipy import stats
from scipy.stats import norm

# Hypoyhesis Part
Ho : µ1(Unit.A) = µ2(Unit.B) Against the Alternative Hypothesis Ha : µ1(Unit.A)  <not equal to>  µ2(Unit.B)

In [19]:
data1=pd.read_csv('D:/EXCELR FILES/Cutlets.csv')

In [20]:
data1.head()

Unit A  Unit B
0  6.8090  6.7703
1  6.4376  7.5093
2  6.9157  6.7300
3  7.3012  6.7878
4  7.4488  7.1522

In [21]:
unitA=pd.Series(data1.iloc[:,0])

In [22]:
unitB=pd.Series(data1.iloc[:,1])

In [23]:
p_value=stats.ttest_ind(unitA,unitB)
p_value

Ttest_indResult(statistic=0.7228688704678061, pvalue=0.4722394724599501)

In [24]:
p_value[1]

0.4722394724599501

# P value for the t-test is found out to be 0.4722 
Conclusion : 
The P value for the two sample t test is found out to be 0.47 so we accept the null hypothesis i.e. there is no significant difference
between the length of cutlets in the two sample units "Unit.A" and "Unit.B" , proved with level of significance 0.05 i.e. confidence level 0.95